superset/utils.py

```diff
@@ -513,6 +513,17 @@ def table_has_constraint(table, name, db):
     return False
 
 
+def upload_file(file):
+    """Takes in the name of the file to be uploaded and uploads that file"""
+
+    from superset import app
```

Comment:
Why are these imports not global?

Reply:
Because there will be cyclical imports if they are global.

Reply:
Because it will cause cyclical import.

Reply:
It would be cleaner to pass the upload_folder from the caller then

Reply:
True. After removing the extra lines, I decided it was best to just call file.save directly as opposed to calling a function with just . one line.